### PR TITLE
Add extract_duration for Swedish

### PR DESCRIPTION
**Pull request by forslund**
#### Description
This adds an initial version of `extract_duration_sv`. The parser is quite simple and will likely need to be extended to handle more complex cases. But this works for many normal sentences.

#### Type of PR
- [ ] Bugfix
- [x] Feature implementation
- [ ] Refactor of code (without functional changes)
- [ ] Documentation improvements
- [ ] Test improvements

#### Testing
Ensure unittests work

### CODE REVIEW

**Comment by ChanceNCounter** [comment]
Cool! Marked for review, but I'm gonna backburner this until we determine if we need to roll back the last two PRs.

**Comment by forslund** [comment]
Hi, just checking if this one is still blocked?

**Comment by ChanceNCounter** [comment]
The TZ fix made it to Chatterbox, but hasn't been backported. I'll see if I can get it over here this weekend.

**Comment by forslund** [comment]
Hi Just checking in on this PR again, is there anything I can do to help with getting it (or something equivalent) into lingua-franca?

File: lingua_franca/lang/parse_sv.py [hunk 1]
```diff
@@ -13,10 +13,145 @@
...
+            simplified.append((number_map[i][0], (number_map[i][1],)))
+
+    if not skip:
+        simplified.append((number_map[-1][0], (number_map[-1][1],)))
+    return simplified
+
+
+def extract_duration_sv(text):
+    """
+    Convert an english phrase into a number of seconds
```
Comment (krisgesling) [inline lingua_franca/lang/parse_sv.py:87]: I haven't considered this before, but we have limits on the length of duration that we support. I wonder if we should add something to the docstrings about that?

Also little typo - "english"

**Review by krisgesling — APPROVED** [review]
Nice stuff. Interesting approach flipping the token order too.

A little question/suggestion below which is a broader question not isolated to this PR - so not a blocker.

File: lingua_franca/lang/parse_sv.py (hunk 1 above)
Comment (forslund) [reply lingua_franca/lang/parse_sv.py:87]: Will fix the typo and look into adding the duration limit

**Comment by forslund** [comment]
Updated the PR, the typo has been taken care of.

**Review by krisgesling — APPROVED** [review]
Yeah nice one. I've done a PR to use this across all other langs too

**Pull request merged**